src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -441,6 +441,9 @@
   <data name="ERR_BadBinaryOps" xml:space="preserve">
     <value>Operator '{0}' cannot be applied to operands of type '{1}' and '{2}'</value>
   </data>
+  <data name="ERR_AmbigBinaryOpsOnUnconstrainedDefault" xml:space="preserve">
+    <value>Operator '{0}' cannot be applied to 'default' and operand of type '{1}' because it is a type parameter that is not known to be a reference type (note: this was incorrectly allowed in an earlier version of the compiler as a null reference comparison)</value>
```

Comment:
Maybe

```suggestion
    <value>Operator '{0}' cannot be applied to 'default' and operand of type '{1}' because it is a type parameter that is not known to be a reference type (note: this was incorrectly allowed in an earlier version of the compiler as a check for null)</value>
```

"as a null reference comparison" sounds a little technical

Reply:
> (note: this was incorrectly allowed in an earlier version of the compiler as a null reference comparison)

I would remove this portion of the message. This is a short term consequence, wouldn't put it into error messages which are long term. Rather I'd make sure we structure our breaking change document so it includes this error codes so it shows up in google searches. We can put the "this used to work" information there.

Reply:
Removed the parenthesized note